config/serving.yaml

```diff
@@ -19,6 +19,10 @@ config:
         - version: 4.14
         - version: 4.11
     "release-next":
+      skipE2EMatches:
```

Comment:
At some point if we add more rules then the config model might have to be revised.

Reply:
+1 on this, but its a good thing that we slowly find all differences